helm/node-exporter-app/templates/psp.yaml

```diff
@@ -2,13 +2,14 @@
 apiVersion: policy/v1beta1
```

Comment:
I'd add `{{- if and (le (int .Capabilities.KubeVersion.Minor) 24) (not .Values.global.podSecurityStandards.enforced) }}`

I don't think PSP is supported anymore after K8s 1.24

Reply:
Yep, makes a lot of sense!

Reply:
I would say "let's just remove the whole thing here", but considering that the whole process is manual and how we sync it from the upstream, every manual change would just make maintenance even harder.

So will leave that type of comments for some future node-exporter v2 or v3 plans 😄